docs/ossm/create-mesh/README.md

```diff
@@ -28,22 +27,19 @@ You can configure each label selector for a variety of use cases, including but
 - Inclusion and/or exclusion of namespaces, for example, all namespaces with label `istio-discovery=enabled` AND label key `app` equal to `helloworld`
 
 #### Using Discovery Selectors to Scope of a Service Mesh
-Assuming you know which namespaces to include as part of the service mesh, as a mesh administrator, you can configure `discoverySelectors` at installation time or post-installation by adding your desired discovery selectors to Istio’s MeshConfig resource. For example, you can configure Istio to discover only the namespaces that have the label `istio-discovery=enabled`.
+Assuming you know which namespaces to include as part of the service mesh, as a mesh administrator, you can configure `discoverySelectors` at installation time or post-installation by adding your desired discovery selectors to Istio’s MeshConfig resource. 
+
+For example, you can configure Istio to discover only the namespaces that have the label `istio-discovery=enabled`.
 
 ##### Prerequisites
 - The OpenShift Service Mesh operator has been installed
 - An Istio CNI resource has been created
-- The `istioctl` binary has been installed on your localhost
 
-1. Create the `istio-system` system namespace:
-    ```bash
-    oc create ns istio-system
-    ```
-1. Label the `istio-system` system namespace:
+1. Add a label to the namespace containing the Istio control plane, for example, the `istio-system` system namespace:
     ```bash
-    oc label ns istio-system istio-discovery=enabled
+    oc label namespace istio-system istio-discovery=enabled
```

Comment:
Do we have to label even the `istio-system` namespace with the same discoverySelector?

Reply:
It was carried over from earlier directions, but made sense to me that istio-system would need to be included, as the control plane will ignore the namespace otherwise. @FilipB, did you confirm that this was needed before? It looks to be documented that way here as well:  https://istio.io/latest/blog/2021/discovery-selectors/, though it looks like they are using `default` for `istio-system`.